templates/strings-structured.stencil

```diff
@@ -0,0 +1,233 @@
+// Generated using SwiftGen, by O.Halligon — https://github.com/AliSoftware/SwiftGen
+
+{% if structuredStrings %}
```

Comment:
Good catch, just added a test that ensures the expected behavior works in cases where there is content within deeper levels but none at the root level (that was the issue). See the message description in 9b3e7d66ea4c2a9f55de07e99cc918a1db8b7b57 for further details.